SimGeneral/TrackingAnalysis/python/trackingParticleNumberOfLayersProducer_cff.py

```diff
@@ -0,0 +1,3 @@
+from SimGeneral.TrackingAnalysis.trackingParticleNumberOfLayersProducer_cfi import *
+from Configuration.StandardSequences.Eras import eras
+eras.fastSim.toModify(trackingParticleNumberOfLayersProducer, simHits=['famosSimHits:TrackerHits'])
```

Comment:
@Dr15Jones Could give a pointer? I wasn't able to find it.

Would the piece above be good-enough interim solution or are there (significantly) better alternatives?